compete.html

```diff
@@ -130,12 +130,12 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Please register your interest below, we'll let you know when formal entries open later this year.
+          Please register your interest below. We'll let you know once places are confirmed. We expect to confirm places at the end of September 2019.
         </p>
         <p class="text-heavy">
           You must be over 18 to run a Student Robotics team and so you must also be over 18 to fill in this form.
         </p>
-        {% include team-signup.html %}
+        <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSfwfZ-tQz31H4-D81cFSmG2p_ciKPpP88MhtXvPmH05yzNzVQ/viewform?embedded=true" width="640" height="1872" frameborder="0" marginheight="0" marginwidth="0">Loadingâ€¦</iframe>
```

Comment:
Can we do something about the placeholder text? It looks like there's an encoding issue there.

Should it be something like `Loading&hellip;`?

Reply:
ah yes, I was fighting with unicode for a while, I think `&hellip;` is the right character :+1:

Reply:
https://github.com/srobo/website/pull/156/commits/71e72ee3f45df136cab0b9742a729aed60fad945